src/audit_payloads.py

```diff
@@ -0,0 +1,194 @@
+# audit_payload.py
+#
+# Writes logs of jobs starting and stopping based on ClassAd updates.
+#
+# This works automatically in a condor-ce if worker nodes have the environment
+#   variable CONDORCE_COLLECTOR_HOST set to point to their site's condor-ce
+#   server, if the jobs are run with condor glideins such as with GlideinWMS.
+#
+# Otherwise a job should use
+#     condor_advertise -pool $CONDORCE_COLLECTOR_HOST:9619 UPDATE_STARTD_AD
+#   at the start of the job and
+#     condor_advertise -pool $CONDORCE_COLLECTOR_HOST:9619 INVALIDATE_STARTD_ADS
+#   at the end of the job, sending to both comannds' standard input at least
+#   these variables with format var = value, string values double-quoted:
+#     Name (string) - name identifying the worker node, typically in the
+#	format user@fully.qualified.domain.name
+#     SlotID (integer) - slot identifier number on the worker node
+#     MyType (string) - required to be set to the value of "Machine"
+#   The condor_advertise at the begining of the message must also contain
+#     GlobalJobId (string) - a globally unique identifier of the job
+#     RemoteOwner (string) - a string identifying the owner of the job
+#   and if the beginning of the message contains any of these they will
+#   also be logged:
+#     ClientMachine (string)
+#     ProjectName (string)
+#     Group (string)
+#     x509UserProxyVOName (string)
+#     x509userproxysubject (string)
+#     x509UserProxyFQAN (string)
+#     x509UserProxyEmail (string)
+#
+# There is one condor-ce configuration variable AUDIT_PAYLOAD_MAX_HOURS,
+#   which is optional and indicates the maximum number of hours any job
+#   is expected to run, default 72 hours (3 days).  After that time the
+#   jobs will stop being tracked, in case a stop message was missed.
+#
+# Written by Dave Dykstra, June 2017
+#
+
+import htcondor
+# I would use the simpler "time" module but it gets a load error
```

Comment:
It was 8.6.3.  All it takes to reproduce is "import time".  When I do that I see this in CollectorLog if D_FULLDEBUG is enabled:
```
07/07/17 12:14:10 Successfully loaded plugin: /usr/libexec/condor/libcollector_python_plugin.so
07/07/17 12:14:10 Python exception occurred when loading module htcondorce.audit_payloads: Traceback (most recent call last):

  File "/usr/lib/python2.7/site-packages/htcondorce/audit_payloads.py", line 1, in <module>
    import time

ImportError: /usr/lib64/python2.7/lib-dynload/timemodule.so: undefined symbol: PyExc_ValueError
```